packages/yew-macro/src/props/component.rs

```diff
@@ -38,6 +38,16 @@ impl Parse for WithProps {
         while !input.is_empty() {
             // no need to check if it's followed by `=` because `with` isn't a special prop
             if input.peek(kw::with) {
+                if let Some((with, expr)) = with_expr {
+                    return Err(syn::Error::new_spanned(
+                        {
+                            let mut res = with.into_token_stream();
+                            expr.to_tokens(&mut res);
+                            res
+                        },
+                        "there are two `with <props>` definitions for this component (note: you can only define `with <props>` once)"
+                    ));
```

Comment:
I like using quote! in cases like these to simplify the code:
https://github.com/yewstack/yew/blob/e6a3ae73018dad2e359efdfde933ed96cd775d58/packages/yew-macro/src/props/prop.rs#L50-L53

```suggestion
                    return Err(syn::Error::new_spanned(
                        quote! { #with#expr },
                        "there are two `with <props>` definitions for this component (note: you can only define `with <props>` once)"
                    ));
```

What do you think?

Reply:
That's neat!